tools/diagrams.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+const globby = require('globby');
+const execa = require('execa')
+const path = require('path')
+const fs = require('fs')
+const chokidar = require('chokidar');
+
+const runMmd = (p) => {
+    const outDir =path.dirname(p).replace('content/', 'static/_gen/diagrams/')
```

Comment:
olint: the olizilla linter
```suggestion
    const outDir = path.dirname(p).replace('content/', 'static/_gen/diagrams/')
```